### ISSUE TITLE
XP PEN STAR G430s not detected by driver

### ISSUE THREAD

**Issue by bbhard** [hawku/TabletDriver#1131]
I tried installing this driver today, but it won't get detected by the driver. Device manager detects it. 
![image](https://user-images.githubusercontent.com/76909216/122553492-6b343500-d040-11eb-82a7-f610be37a4af.png)

I tried running the guy as administrator, but it wouldn't fix the issue. What else can I do to fix this?

## Hardware/software information
Windows Version: 21H1

Tablet Driver version: 0.2.3

Tablet model: xp pen g430s

**Comment by SnowLire** [comment]
#1087

**Comment by Kiried-Menacing** [comment]
Have you ran the install_vmulti_driver.bat file or
install_vmulti_driver32.bat
The "install_vmulti_driver.bat" is for 64 bit systems
The  "install_vmulti_driver32.bat" is for 32 bit systems

**Comment by SnowLire** [comment]
no, just, uninstall hawku. the vmulti included is broken on 21H1, use OpenTabletDriver
